apps/src/codebridge/hooks/useInitialSources.ts

```diff
@@ -41,11 +41,11 @@ export const useInitialSources = (
 ) => {
   const isStartMode = getAppOptionsEditBlocks() === START_SOURCES;
   const exemplarSources = levelProperties.exemplarSources as MultiFileSource;
+  const startSources = levelProperties.startSources as MultiFileSource;
```

Comment:
FYI @molly-moen - minor change as a result of changing the type of `startSources` to `Source` (trying to see if we can unify around all labs using the same `startSources` field where possible)

Also note that this means that Lab2 Dance will _not_ respect the `startBlocks` level property (@FYI @mikeharv @breville) - again making an intentional split here to move away from Blockly XML and unify field names across lab2 labs.